### PR TITLE
New website design

### PR DESCRIPTION
**Pull request by cassidoo**

### CODE REVIEW

**Comment by sallychenyx9** [comment]
- Links will open in a new tab

- "What it's like to work with us":  [](https://codepen.io/flatpixels/pen/MWYEqdy) have a horizontal scrolling animation for overflowing contents (testimonials and logos should both be one line)

- 3.More spacing above and below the "What it's like to work with us" section. (visually people shouldn't be able to see two whole sections at the same time)

- Nit: I can't close the toggles in the Q&A section

**Comment by GradySalzman** [comment]
> * Links will open in a new tab
> * "What it's like to work with us":   have a horizontal scrolling animation for overflowing contents (testimonials and logos should both be one line)
> * 3.More spacing above and below the "What it's like to work with us" section. (visually people shouldn't be able to see two whole sections at the same time)
> * Nit: I can't close the toggles in the Q&A section

All these comments have been addressed and fixed

File: src/components/Accordion.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ function Accordion({ data }) {
 							className={`${isActive ? styles.expanded : ""} ${
 								styles.title
 							}`}
-							onClick={() => setActiveIndex(index)}
+							onClick={() => activeIndex === index ? setActiveIndex(null) : setActiveIndex(index)}
```
Comment (cassidoo) [inline src/components/Accordion.jsx:17]: Nice, my bad

**Review by jlin99 — COMMENTED** [review]
i clicked all the buttons and here's what i found

- clicking `partners` brings me to like halfway where it should and stutters a lot
- clicking `how it works` at the bottom of the page brings me nowhere

everything else looks good tho!!

**Comment by sallychenyx9** [comment]
- Reduce spacing between the hero section and "How Contenda Works" - hero should be a bit lower on the page, and people should be able to see a litttttttle bit of "How Contenda Works" to indicate that there's more content 
-Add a bit more spacing between the hero title, description and buttons (on Figma I had 108px spacing)
-Fonts seem not up to date? We are using Inter but the inspect tool shows that the current fonts are Archivo - check Figma for the latest font updates
-"What it's like to work with us" - each quote box should have a bit more padding, and the boxes should be a bit closer to each other
-"FAQ" section - first question should be open by default 

MOBILE SPECIFIC COMMENTS:
-(this is a question) - are we getting rid of the three dots in "What it's like to work with us" ? 
-A bit more padding around the "FAQ" section - currently the text almost touches the edge of the screen

**Comment by sallychenyx9** [comment]
Top nav bar: items should be capitalized (refer to Figma design)

**Comment by sallychenyx9** [comment]
IM SO SORRY, FEW MORE COMMENTS ON LINKS

- "What it's like to work with us" quote boxes - logo should link to the corresponding companies
- "Is Contenda compatible with other video platforms besides YouTube?" (FAQ)- "Twitch, Mux, and YouTube" should have their corresponding links
- "How can I contact Contenda support for assistance or questions?" (FAQ) - socials should have their corresponding links
- Footer: add logos for socials (refer to Figma design)

**Comment by GradySalzman** [comment]
> i clicked all the buttons and here's what i found
> 
> * clicking `partners` brings me to like halfway where it should and stutters a lot
> * clicking `how it works` at the bottom of the page brings me nowhere
> 
> everything else looks good tho!!

TY! Both of these have been fixed

**Comment by GradySalzman** [comment]
> IM SO SORRY, FEW MORE COMMENTS ON LINKS
> 
> * "What it's like to work with us" quote boxes - logo should link to the corresponding companies
> * "Is Contenda compatible with other video platforms besides YouTube?" (FAQ)- "Twitch, Mux, and YouTube" should have their corresponding links
> * "How can I contact Contenda support for assistance or questions?" (FAQ) - socials should have their corresponding links
> * Footer: add logos for socials (refer to Figma design)

All these comments a have been fixed!

**Comment by sallychenyx9** [comment]
- Hero section - body ("Unlock content from the rest of your team...") font should also be 16px
- Footer - © 2023 Contenda Inc should be left aligned with the Contenda logo

**Comment by GradySalzman** [comment]
> * Hero section - body ("Unlock content from the rest of your team...") font should also be 16px
> * Footer - © 2023 Contenda Inc should be left aligned with the Contenda logo

Fixed!

**Pull request merged**